code/src/App.js

```diff
@@ -1,9 +1,78 @@
-import React from 'react';
+
+import React, { useState } from 'react';
+import { StartButton } from 'components/StartButton';
+import { Result } from 'components/Result';
+import { WeeklyTraining } from 'components/WeeklyTraining';
+import { PreviousButton } from 'components/PreviousButton';
+import { NextButton } from 'components/NextButton';
+import { SubmitButton } from 'components/SubmitButton';
+import { Intro } from 'components/Intro';
+import { AgeGroup } from './components/AgeGroup';
+import { YesNoQuestion } from './components/YesNoQuestion';
+import { SportType } from './components/SportType';
 
```

Comment:
It's really interesting to see how you have made components for the buttons. I did the buttons in a very basic way, without even one component, so thank you for showing me something different. I will probably try this next time. :)